### PR TITLE
Responsive Faq page

### PR DESCRIPTION
**Pull request by NicholasAntonov**

### CODE REVIEW

**Comment by eshellman** [comment]
This works. But the default text is too small to read on iPhone; which is a problem for all the pages.

File: frontend/templates/basedocumentation.html [hunk 1]
```diff
@@ -8,6 +8,7 @@
 {% block extra_extra_head %}
 <link type="text/css" rel="stylesheet" href="{% sass_src 'scss/landingpage4.scss' %}" />
 <link type="text/css" rel="stylesheet" href="{% sass_src 'scss/book_panel2.scss' %}" />
+<link type="text/css" rel="stylesheet" href="{% sass_src 'scss/faq.scss' %}" />
```
Comment (eshellman) [inline frontend/templates/basedocumentation.html]: better for load times to put all the css in one file

File: static/js/sitewide1.js [hunk 2]
```diff
@@ -11,57 +11,6 @@ $j(document).ready(function() {
...
-
-                if (vars.length>1){
-                    next=vars[1];
-                    if(next!='') {
-                        next = next.replace(/[\x22\x27\x3c\x3e]/g,'');
-                        $j.cookie('next', next, {path: '/'});
-                    }
-                }
-            }
-            // fade-out rest of page elements on expand
```
Comment (eshellman) [inline static/js/sitewide1.js]: I don't think this code ever worked

File: static/scss/sitewide4.scss [hunk 3]
```diff
@@ -569,67 +563,6 @@ a#readon {
...
-    }
-
-    .jsmod-content {
-        border: solid 1px $pale-blue;
-        @include border-radius(0, 0, 10px, 10px);
-    }
-
-    ul {
-        &.level1 > li > a, &.level1 > li > span {
-            border-bottom:1px solid $pale-blue;
```
Comment (eshellman) [inline static/scss/sitewide4.scss]: the 1 pixel borders are nice on desktop at least - would it be hard to keep them?

**Comment by NicholasAntonov** [comment]
In response to the small text @eshellman , that was only appearing that way due to the meta tag not being set yet in this branch despite being fixed in another branch. I merged newfoundation into this branch so the final result should be what you see on checkout now.

**Comment by eshellman** [comment]
When I try to run this, I get a lot of 404 errors

[01/May/2018 08:22:47] "GET /static/scss/font-awesome.min.css HTTP/1.1" 404 1692
[01/May/2018 08:22:48] "GET /static/scss/$%7Bimage-base%7Dbg.png HTTP/1.1" 404 1693
[01/May/2018 08:22:50] "GET /static/scss/font-awesome.min.css HTTP/1.1" 404 1692
[01/May/2018 08:22:50] "GET /static/scss/$%7Bimage-base%7Dbg.png HTTP/1.1" 404 1693
WARNING: null was passed to rem-calc(), which is not a number.
Backtrace:
	static/scss/foundation/scss/util/_unit.scss:78, in function `-zf-to-rem`
	static/scss/foundation/scss/util/_unit.scss:47, in function `rem-calc`
	static/scss/foundation/scss/xy-grid/_cell.scss:84, in function `if`
	static/scss/foundation/scss/xy-grid/_cell.scss:84, in mixin `-xy-cell-properties`
	static/scss/foundation/scss/xy-grid/_cell.scss:167, in mixin `xy-cell-static`
	static/scss/foundation/scss/xy-grid/_classes.scss:275, in mixin `xy-vertical-grid-classes`
	static/scss/foundation/scss/xy-grid/_classes.scss:470, in mixin `foundation-xy-grid-classes`
	static/scss/foundation/scss/foundation.scss:84, in mixin `foundation-everything`
	static/scss/global.scss:5

WARNING: Contrast ratio of #fefefe on #8dc63f is pretty bad, just 2
Backtrace:
	static/scss/foundation/scss/util/_color.scss:78, in function `color-pick-contrast`
	static/scss/foundation/scss/components/_button.scss:149, in mixin `button-style`
	static/scss/foundation/scss/components/_button.scss:311, in mixin `foundation-button`
	static/scss/foundation/scss/foundation.scss:92, in mixin `foundation-everything`
	static/scss/global.scss:5

WARNING: Contrast ratio of #fefefe on #ffae00 is pretty bad, just 1.8
Backtrace:
	static/scss/foundation/scss/util/_color.scss:78, in function `color-pick-contrast`
	static/scss/foundation/scss/components/_button.scss:149, in mixin `button-style`
	static/scss/foundation/scss/components/_button.scss:311, in mixin `foundation-button`
	static/scss/foundation/scss/foundation.scss:92, in mixin `foundation-everything`
	static/scss/global.scss:5

WARNING: Contrast ratio of #fefefe on #8dc63f is pretty bad, just 2
Backtrace:
	static/scss/foundation/scss/util/_color.scss:78, in function `color-pick-contrast`
	static/scss/foundation/scss/components/_button.scss:211, in mixin `button-disabled`
	static/scss/foundation/scss/components/_button.scss:332, in mixin `foundation-button`
	static/scss/foundation/scss/foundation.scss:92, in mixin `foundation-everything`
	static/scss/global.scss:5

WARNING: Contrast ratio of #fefefe on #ffae00 is pretty bad, just 1.8
Backtrace:
	static/scss/foundation/scss/util/_color.scss:78, in function `color-pick-contrast`
	static/scss/foundation/scss/components/_button.scss:211, in mixin `button-disabled`
	static/scss/foundation/scss/components/_button.scss:332, in mixin `foundation-button`
	static/scss/foundation/scss/foundation.scss:92, in mixin `foundation-everything`
	static/scss/global.scss:5

WARNING: Contrast ratio of #fefefe on #8dc63f is pretty bad, just 2
Backtrace:
	static/scss/foundation/scss/util/_color.scss:78, in function `color-pick-contrast`
	static/scss/foundation/scss/components/_button.scss:149, in mixin `button-style`
	static/scss/foundation/scss/components/_button-group.scss:215, in mixin `foundation-button-group`
	static/scss/foundation/scss/foundation.scss:97, in mixin `foundation-everything`
	static/scss/global.scss:5

WARNING: Contrast ratio of #fefefe on #ffae00 is pretty bad, just 1.8
Backtrace:
	static/scss/foundation/scss/util/_color.scss:78, in function `color-pick-contrast`
	static/scss/foundation/scss/components/_button.scss:149, in mixin `button-style`
	static/scss/foundation/scss/components/_button-group.scss:215, in mixin `foundation-button-group`
	static/scss/foundation/scss/foundation.scss:97, in mixin `foundation-everything`
	static/scss/global.scss:5

File: static/scss/sitewide4.scss (hunk 3 above)
Comment (eshellman) [reply static/scss/sitewide4.scss]: when in doubt, don't change things - they make returning users comfortable

**Pull request merged**